examples/react-nodejs/server.js

```diff
@@ -1,14 +1,18 @@
-const express = require('express');
+import express from 'express';
+import { join, dirname } from 'path';
+import { fileURLToPath } from 'url';
+import Markdoc from '@markdoc/markdoc';
+import callout from './schema/Callout.markdoc.js';
+import heading from './schema/heading.markdoc.js';
+import { createContentManifest } from './createContentManifest.js';
+
 const app = express();
-const path = require('path');
 
-const Markdoc = require('@markdoc/markdoc');
-const callout = require('./schema/callout.markdoc');
-const heading = require('./schema/heading.markdoc');
-const { createContentManifest } = require('./createContentManifest');
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 
-const PORT = 4242;
-const CONTENT_DIR = path.join(__dirname, 'content');
+const PORT = 5001;
```

Comment:
```suggestion
const PORT = 4242;
```
Thoughts on keeping port 4242 similar to the legacy app?

Reply:
Works for me, I just didn't notice for whatever reason!